hw/top_earlgrey/data/ip/chip_rv_dm_testplan.hjson

```diff
@@ -0,0 +1,291 @@
+// Copyright lowRISC contributors.
+// Licensed under the Apache License, Version 2.0, see LICENSE for details.
+// SPDX-License-Identifier: Apache-2.0
+{
+  name: rv_dm
+  testpoints: [
+    // RV_DM (JTAG) tests:
+    {
+      name: chip_jtag_csr_rw
+      desc: '''
+            Verify accessibility of all the CSRs in the chip over JTAG.
+
+            - Shuffle the list of CSRs first to remove the effect of ordering.
```

Comment:
Is that literally *all* of the CSRs? How do we handle CSRs that have nontrivial side-effects?